datafusion/core/tests/parquet/arrow_statistics.rs

```diff
@@ -159,9 +159,9 @@ impl TestReader {
 }
 
 /// Defines a test case for statistics extraction
-struct Test {
+struct Test<'a> {
     /// The parquet file reader
-    reader: ParquetRecordBatchReaderBuilder<File>,
+    reader: &'a ParquetRecordBatchReaderBuilder<File>,
```

Comment:
the change is to reuse the reader rather than recreating it each time

Reply:
👍